glean-core/ios/Glean/Debug/GleanDebugTools.swift

```diff
@@ -22,7 +22,7 @@ class GleanDebugUtility {
     /// There are 3 available commands that you can use with the Glean SDK debug tools
     ///
     /// - `logPings`: If "true", will cause pings that are submitted successfully to also be echoed to the device's log
-    /// - `tagPings`:  This command expects a string to tag the pings with and redirects them to the Glean Debug View
+    /// - `debugViewTag`:  This command expects a string to tag the pings with and redirects them to the Debug View
     /// - `sendPing`: This command expects a string name of a ping to force immediate collection and submission of.
```

Comment:
Is there a plan to support the source tags on iOS, too?

Reply:
(I'm fine with not doing it right now)

Reply:
Yup, [filed this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1654197) for the implementation work there. I might need somebody else's help :)